test/server/GraknRunnerBase.java

```diff
@@ -71,15 +71,14 @@ public GraknRunnerBase(File distributionArchive, boolean debug) throws Interrupt
         this.distributionArchive = distributionArchive;
         distributionArchiveFormat = distributionFormat(distributionArchive);
         distributionDir = distributionTarget(distributionArchive);
-        dataDir = distributionDir.resolve("server").resolve("data");
```

Comment:
Why are you moving this down to a method to be initialised later when the value never changes, @vmax ?

Reply:
@haikalpribadi not only is it moved, but also changed: previously we resolved the path against `distributionDir`, which is not correct - we should resolve it against the _unpacked_ dir (`graknPath`)

Reply:
I think it's because the value depends on a path that can only be determined later, once the distribution archive has been unpacked: https://github.com/graknlabs/common/blob/235a13273426e170e1e7edebe04624ba2e97f61c/test/server/GraknRunnerBase.java#L178-L180 @haikalpribadi 

right @vmax ?

Reply:
@lolski this is correct

Reply:
Actually, you can initialise `dataDir` in the constructor if you return the path of the distribution @vmax . Let's do that since it's cleaner.

Reply:
let's do what ganesh suggested first

Reply:
@haikalpribadi @lolski addressed in https://github.com/graknlabs/common/pull/82/commits/d8e0c743a1a5cf9111b064ae93be44714ae13148